x-pack/plugins/logstash/server/routes/upgrade/execute.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { ICustomClusterClient, IRouter } from 'src/core/server';
+import { licenseCheckerRouteHandlerWrapper } from '../../../../licensing/server';
+
+import { INDEX_NAMES } from '../../../common/constants';
+import { checkLicense } from '../../lib/check_license';
+
+export function registerUpgradeRoute(router: IRouter, esClient: ICustomClusterClient) {
+  router.post(
+    {
+      path: '/api/logstash/upgrade',
```

Comment:
Just because it was `x-pack/legacy/plugins/logstash/server/routes/api/upgrade/register_execute_route.js` before. I didn't want to change the intention of the original author. Not strong preference here

Reply:
It's been a couple years but IIRC the team at the time had some naming convention here based on the HTTP method of the route. I think that's where the "execute" is coming from. I have no strong preferences here either; feel free to change it to something that makes more sense to you or leave it as is!

Reply:
let's keep it that way. this PR is only about migrating to NP. Marking as resolved.

Reply:
ok, I will rename